app/src/main/java/protect/card_locker/LoyaltyCard.java

```diff
@@ -34,12 +34,14 @@ public class LoyaltyCard implements Parcelable {
     public final long lastUsed;
     public int zoomLevel;
 
+    public int usage_number;
```

Comment:
Can't this be final like the rest?

Also, please use camel-case like everywhere else.